### PR TITLE
Update redux-form_v5.x.x.js

### PR DESCRIPTION
**Pull request by lvdang**
Change to be compatible with latest flow v58.0

### CODE REVIEW

**Comment by lvdang** [comment]
@GAntoine - thanks. That is my mistake.

**Comment by lvdang** [comment]
@GAntoine - how do I create a subFolder? I only have option to create file?

**Comment by lvdang** [comment]
@GAntoine - I created this file: https://github.com/flowtype/flow-typed/pull/1502

**Comment by GAntoine** [comment]
Give me a couple minutes on this PR, I'll fix it up and get it to where you can fix the bugs. I closed the other one since both changes can be done here.

**Comment by GAntoine** [comment]
@lvdang Ok, I've modified the folder structure to something usable. However the definition you wrote for 53+ are still broken, so you'll need to address those.

**Comment by lvdang** [comment]
@GAntoine - thanks. let me take a look

**Comment by lvdang** [comment]
@GAntoine  - this is my first flow type checkin. Hopefully I am able to fix the issues.

**Comment by lvdang** [comment]
@GAntoine  - got a question about one of the fail error

// With React documentation SyntheticEvennt<> is used for v53. Could it be that the error is with this declaration?

 (props.onChange: (v: SyntheticEvent) => mixed);
                                 ^^^^^^^^^^^^^^ SyntheticEvent. Application of polymorphic type needs <list of 0-1 arguments>.

**Comment by GAntoine** [comment]
Try changing `SyntheticEvent<>` to `SyntheticEvent<*>`

**Comment by GAntoine** [comment]
Looks like the only remaining issue is with `React.Node`. Unfortunately that's not my field of expertise.

**Comment by lvdang** [comment]
@GAntoine  we are very close ... one more bug to zap

File: definitions/npm/redux-form_v5.x.x/flow_v0.53.x-/redux-form_v5.x.x.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  declare export type FormConfig = {
+    fields: Array<string>,
+    form: string,
+    alwaysAsyncValidate?: boolean,
+    asyncBlurFields?: Array<string>,
+    asyncValidate?: (values: Object, dispatch: Function, props: Object) => Promise<void>,
+    destroyOnUnmount?: boolean,
+    formKey?: string,
+    getFormState?: (state: Object, reduxMountPoint: string) => mixed,
+    initialValues?: { [field: string]: string },
```
Comment (villesau) [inline definitions/npm/redux-form_v5.x.x/flow_v0.53.x-/redux-form_v5.x.x.js]: initialValues can be also nested object, at least thats how we use it in our project.

**Pull request closed without merging**